examples/websockets-realtime-voting/e2e-tests/playwright.config.ts

```diff
@@ -1,5 +1,6 @@
 import { defineConfig, devices } from "@playwright/test";
 
+const WASP_APP_RUNNER_CLI = process.env.WASP_APP_RUNNER_CLI ?? "run-wasp-app";
```

Comment:
```suggestion
const WASP_APP_RUNNER_CLI_CMD = process.env.WASP_APP_RUNNER_CLI ?? "run-wasp-app";
```

To stay consistent with `WASP_CLI_CMD` and since it's a command 😄